### PR TITLE
Home

### PR DESCRIPTION
**Pull request by norasadder**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -23,14 +26,47 @@
...
 </header>
 
 
-<section id = "Home">
+
+<main>  
+
+<section class = "Home" id = "Home">
+
+<div style="width: 5%;">
```
Comment (TamerHN) [inline index.html]: avoid inline styles to avoid noise between the CSS file and the inline styles

File: index.html [hunk 2]
```diff
@@ -23,14 +26,47 @@
...
+
+<div class = "Home-text" style="width: 45%;">
+    <h1>
+        Hello, I'm <span> Nora Sadder </span> 
+    </h1>
+    <h2>
+        Frontend Developer
+    </h2>
+    <h3>
+        Eperienced in web development, creating delightful interfaces with user experience.
```
Comment (TamerHN) [inline index.html]:
```suggestion
        Experienced in web development, creating delightful interfaces with user experience.
```

File: index.css [hunk 3]
```diff
@@ -1,16 +1,19 @@
...
+    color: var(--DarkPurple);
+    text-decoration: none;
 }
 
 html{
-    font-size: 20px;
+    font-size: 1.5vw;
 }
 
 body{
```
Comment (TamerHN) [inline index.css]: remove the default `margin` of the body


```css
margin: 0;
```

File: index.html [hunk 4]
```diff
@@ -23,14 +26,47 @@
...
+
+<div style="width: 5%;">
+
+</div>
+
+<div style="width: 40%;">
+    <img src = "images/personal.png" alt="Nora's Photo" width = "100%" height="auto">
+</div>
+
+<div style="width: 5%;">
```
Comment (TamerHN) [inline index.html]: empty divs mustn't be used for spacing.

**Comment by TamerHN** [comment]
Here how you should organize your page.
image

1. Add a variable for header height
```css
:root {
  --header-height: 50px
}
header {
...
height: var(--header-height);
...
}
```
2. Set the width for the Home to (80vw) - (10vh padding from each side).
3. Calculate the height of the home screen to be exactly the height of the screen you have.
```css
.Home {
  display: flex;
  justify-content: center;
  align-items: center;
  width: 80vw;
  padding:var(--header-height) 10vw 0 10vw;
  height: calc(100vh - var(--header-height));
}
```
4. Remove empty divs you added for spacing, remove width property from inline styles. 
Add `flex: 1` for each of the divs inside home `section`.

```css
.Home > div {
    flex: 1;
}
```

**Review by TamerHN — CHANGES_REQUESTED** [review]

File: index.html (hunk 4 above)
Comment (norasadder) [reply index.html]: Yes, I removed them before but in another branch

**Pull request merged**